### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by luisvgs**
Provided basic css selectors concepts with examples.

### CODE REVIEW

File: 3_css/readme.md [hunk 1]
```diff
@@ -86,13 +86,49 @@ Comments are intended for better code comprehension, this are pieces of the code
...
 ```
 ## CSS basic selectors 
-    CSS selectors are used to select the HTML element that we want to style.
-    Basic selectors in CSS are the following:
+Selectors lets you modify individual elements in our HTML file. Here are some useful selector you should know:
     
-    1) Element selector: Selects HTML element using their name.
-    2) ID selector: Selects HTML element using their id attribute. Usage: Hash(#) followed by id.
-    3) Class selector: Selects HTML element using their class name. Usage: Period(.) followed by class name.
-    4) Universal selector: Selects all HTML elements in the page. Usage: Asterisk(*) is used as universal selector
```
Comment (DanielOrmeno) [inline 3_css/readme.md]: Please don't remove this content, merge your changes instead so that your contributions does not remove these changes.

**Review by DanielOrmeno — CHANGES_REQUESTED** [review]
Thank you for your contribution! this is great. Please check my comments as I have requested some changes.

File: 3_css/readme.md (hunk 1 above)
Comment (luisvgs) [reply 3_css/readme.md]: Hello Daniel! I followed your instructions. It's okay now? Let me know! Thank you.

Comment (DanielOrmeno) [reply 3_css/readme.md]: Thank you!

**Pull request merged**